crates/api_common/src/utils.rs

```diff
@@ -614,7 +626,11 @@ pub async fn send_new_report_email_to_admins(
   let reports_link = &format!("{}/reports", settings.get_protocol_and_hostname(),);
 
   for admin in &admins {
-    let email = &admin.local_user.email.clone().expect("email");
+    let email = &admin
+      .local_user
+      .email
+      .clone()
+      .ok_or(LemmyErrorType::EmailRequired)?;
```

Comment:
This and the previous one seem wrong, Lemmy shouldnt throw an error just because there is an admin with no email configured. It should simply skip that admin instead.

Reply:
This list is already filtered to admins with emails only, so I know its there. I could either use a `LemmyError::Unreachable` or `EmailRequired`.

Reply:
Actually I'll use an if let, that's safer and avoids this entirely.